package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "watson-speech-to-text-utils",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
We change the `package.json` using [`npm version`](https://docs.npmjs.com/cli/version), we don't do it manually.